webapp/src/main/resources/db/migration/V62__branch_notifiers.sql

```diff
@@ -0,0 +1,3 @@
+alter table branch_notification add column notifier_id varchar(255);
+create table branch_notifiers (branch_id bigint not null, notifiers varchar(255)) engine=InnoDB;
```

Comment:
Was the "engine=InnoDB" part left in by accident?
We don't specify the underlying storage engine anywhere else - outside of some Quartz tables and spring session table. (Do you have any information if they're required there?)